Dave 21

crystal95 opened the pull request.

crystal95 committed:
Initial commit

crystal95 committed:
resizing each plot with window resize

crystal95 committed:
Managing error due to empty form field

crystal95 committed:
changed filename

pbalm committed:
Adding file with the requested order of the columns

pbalm committed:
Adding __pycache__ to global gitignore

pbalm commented on src/main/js/electron/templates/index.html:
```diff
@@ -3,39 +3,45 @@
-  });
-});
-</script>
-<script>
```
Why did you remove this section? With this removed, we don't see the filename in the "load spectrum" textbox anymore after selecting, right?

pbalm replied:
Added again in PR for DAVE-22

pbalm commented:
Looks ok except for two things, the filename isn't showing anymore after loading (see comment above) and I think you're using the wrong column order in your input data but I'm not sure it matters. Anyway it shouldn't be hard to fix, let's see what Simone says.

crystal95 commented:
Ah , I added it again in DAVE-22 :) , I removed it for checking something , any ways it will be added back when we merge DAVE-22 ........... Also I will reorder the columns to suit the test file I will get :)

crystal95 committed:
Reordered columns

crystal95 committed:
Re-added File Choser Text

crystal95 committed:
Managing conflicts

pbalm committed:
Merging master and resolving conflicts
